### PR TITLE
Add method of `Card` to describe its suit and rank

### PR DESCRIPTION
**Pull request by azriel1rf**
implementation of #37

### Python:
- `card.py`: the main update for Python
- `evaluator.py`: eliminate code duplication with `card.py` using its code change
- `test_card.py`: create new file to test `card.py`
- `test_evaluator.py`: add test for `evaluate_cards()` about integration with `Card`

### C++:
- `card.h`: the main update for C++

### Others:
- `ci.yml`: considering Python tests becoming across two files, test multiple files in `test` directory at once with `unittest`'s feature. checked that this worked.
- `README.md`: update about that

### CODE REVIEW

File: cpp/include/phevaluator/card.h [hunk 1]
```diff
@@ -23,6 +23,23 @@
...
+const static std::unordered_map<char, int> rankMap = {
+  {'2', 0}, {'3', 1}, {'4', 2}, {'5', 3},
+  {'6', 4}, {'7', 5}, {'8', 6}, {'9', 7},
+  {'T', 8}, {'J', 9}, {'Q', 10}, {'K', 11}, {'A', 12},
+};
+const static std::unordered_map<char, int> suitMap = {
+  {'C', 0}, {'D', 1}, {'H', 2}, {'S', 3},
+  {'c', 0}, {'d', 1}, {'h', 2}, {'s', 3},
+};
+const static std::unordered_map<int, char> rankReverseMap = {
```
Comment (HenryRLee) [inline cpp/include/phevaluator/card.h:35]: Since the indexes are ranged from 0 to 12, do you think we can replace the `unordered_map` with `array`?

**Review by HenryRLee — CHANGES_REQUESTED** [review]
Thanks for your awesome work. I left one change request in the `card.h` header file.

File: cpp/include/phevaluator/card.h (hunk 1 above)
Comment (azriel1rf) [reply cpp/include/phevaluator/card.h]: Good idea. I modified it.

**Review by HenryRLee — APPROVED** [review]

**Comment by HenryRLee** [comment]
That's great. Thank you.

**Pull request merged**